test/check-application

```diff
@@ -175,6 +175,16 @@ class TestApplication(testlib.MachineCase):
         self.system_images_count = int(self.execute(True, "podman images -n | wc -l").strip())
         self.user_images_count = int(self.execute(False, "podman images -n | wc -l").strip())
 
+    def tearDown(self):
```

Comment:
addcleanup is called after tearDown so this sounds sensible!

Reply:
Yep, I learned the same in https://github.com/cockpit-project/cockpit/commit/163555f90dc2872be715da073ae468c9b77867cf